tests/benchmarks/power_consumption/flash/testcase.yaml

```diff
@@ -0,0 +1,20 @@
+common:
```

Comment:
please add `integration_platforms` section

Reply:
added for all tests